packages/main/src/modules/config.ts

```diff
@@ -1,30 +1,38 @@
 import path from 'node:path';
-import { FileSystemStorage } from '/shared/types/storage';
+import { FileSystemStorage, type IFileSystemStorage } from '/shared/types/storage';
 import { SETTINGS_DIRECTORY, CONFIG_FILE_NAME, getFullScenesPath } from '/shared/paths';
-import { DEFAULT_CONFIG, mergeConfig } from '/shared/types/config';
+import { DEFAULT_CONFIG, mergeConfig, type Config } from '/shared/types/config';
 import { getUserDataPath } from './electron';
+import { waitForMigrations } from './migrations';
+import log from 'electron-log/main';
 
 export const CONFIG_PATH = path.join(getUserDataPath(), SETTINGS_DIRECTORY, CONFIG_FILE_NAME);
-const storage = await FileSystemStorage.getOrCreate(CONFIG_PATH);
 
-// Initialize with default values if empty
-const existingConfig = await storage.get<Record<string, any>>('');
```

Comment:
This line was the bug, this always returns `undefined`, and causes the default config to be applied over the existing one.